BinarySearchTree.h

```diff
@@ -36,7 +36,7 @@ class BinarySearchTree
     bool isBalancedInternal(TreeNode *root);
     int heightInternal(TreeNode *root);
     void PrintLevelWiseInternal(TreeNode* root,int Level);
-
```

Comment:
It is nice to not delete the line so that we are consistent with previous function declarations.
Because other function declarations do not have an extra line between them.